test/integration/transport_scene_manager.cc

```diff
@@ -260,6 +261,5 @@ TEST(TransportSceneManagerTest, GZ_UTILS_TEST_ENABLED_ONLY_ON_LINUX(Config))
 
   win->QuickWindow()->close();
   engine->DestroyScene(scene);
-  EXPECT_TRUE(rendering::unloadEngine(engine->Name()));
```

Comment:
?

Reply:
Unloading the scene is what causes the issues.